docker-compose.yml

```
@@ -0,0 +1,38 @@
+version: "3.8"
+
+services:
+  mysqldb:
+    platform: linux/arm64
+    image: mysql:8
+    restart: unless-stopped
+    env_file: ./.env
+    environment:
+      - MYSQLDB_USER=${MYSQLDB_USER}
+      - MYSQLDB_PASSWORD=${MYSQLDB_PASSWORD}
+      - MYSQLDB_ROOT_PASSWORD=${MYSQLDB_ROOT_PASSWORD}
+      - MYSQLDB_DB=${MYSQLDB_DATABASE}
```

docker is sensitive to the naming
try these
```suggestion
      - MYSQL_ROOT_PASSWORD=${MYSQLDB_PASSWORD}
      - MYSQL_DATABASE=${MYSQLDB_DATABASE}
```